apps/web/src/pages/brand/tabs/v2/BrandingForm.tsx

```diff
@@ -0,0 +1,232 @@
+                            <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                            <Text color={colors.B70}>Upload</Text>
+                          </Stack>
+                        ) : (
+                          <img
+                            data-test-id="logo-image-wrapper"
+                            src={field.value}
+                            className={css({
+                              minHeight: '6.25rem',
+                              minWidth: '6.25rem',
+                              objectFit: 'contain',
+                            })}
+                            alt="avatar"
+                          />
+                        )}
+                      </div>
+                    </Dropzone>
+                  </div>
+                )}
+                control={control}
+                name="brandLogoUrl"
+              />
+            </Stack>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput label="Brand color" data-test-id="brand-color-picker" disallowInput={false} {...field} />
+                )}
+                control={control}
+                name="brandColorValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <Select
+                    label="In-App font family"
+                    placeholder="Select a font family"
+                    data={[
+                      'inherit',
+                      'Fira Code',
+                      'Roboto',
+                      'Montserrat',
+                      'Open Sans',
+                      'Lato',
+                      'Nunito',
+                      'Oswald',
+                      'Raleway',
+                    ]}
+                    data-test-id="font-family-selector"
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontFamilyValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput
+                    label="In-App font color"
+                    data-test-id="font-color-picker"
+                    disallowInput={false}
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontColorValue"
+              />
+            </div>
+          </Stack>
+
+          <div
+            className={css({
+              textAlign: 'right',
+              marginTop: '60px',
+            })}
+          >
+            <Button submit loading={isLoading} data-test-id="submit-branding-settings" px="4rem">
```

Comment:
```suggestion
            <Button submit loading={isLoading} data-test-id="submit-branding-settings" **px="4rem"**>
```
✏ suggestion: ‏ We should reconsider this value and confirm with Nikolay‏